write-fonts/src/tables/cmap.rs

```diff
@@ -181,20 +177,11 @@ mod tests {
                     cmap4.range_shift()
                 )
             );
-            assert_eq!(
-                vec![10u16, 30u16, 153u16, 0xffffu16],
-                to_vec(cmap4.start_code())
-            );
-            assert_eq!(
-                vec![20u16, 90u16, 480u16, 0xffffu16],
-                to_vec(cmap4.end_code())
-            );
+            assert_eq!(cmap4.start_code(), &[10u16, 30u16, 153u16, 0xffffu16]);
+            assert_eq!(cmap4.end_code(), &[20u16, 90u16, 480u16, 0xffffu16]);
             // The example starts at gid 1, we're starting at 0
-            assert_eq!(vec![-10i16, -19i16, -81i16, 1i16], to_vec(cmap4.id_delta()));
-            assert_eq!(
-                vec![0u16, 0u16, 0u16, 0u16],
-                to_vec(cmap4.id_range_offsets())
-            );
```

Comment:
I'm not sure I follow why you need to "fix" this, wouldn't the test be completely fine as it was before still?

Reply:
The old version is functionally equivalent, just unnecessarily verbose.

Reply:
yes, that's the annoyance. If you prefer having that ordering over having less code then by all means you should keep it as-is and we can close this 🤷

Reply:
My personal inclination would be to merge partialeq but not update the test. It's also fine to just update the test :)

Reply:
What do you mean 'merge PartialEq'?

Reply:
Basically revert write-fonts/src/tables/cmap.rs. I think I misread this PR as _adding_ the partialeq as opposed to just a comment.

The order of asserts is not so important so I'm also entirely fine if you just merge as-is